subiquity/common/filesystem/labels.py

```diff
@@ -336,3 +342,14 @@ def _for_client_partition(partition, *, min_size=0):
 @for_client.register(gaps.Gap)
 def _for_client_gap(gap, *, min_size=0):
     return types.Gap(offset=gap.offset, size=gap.size, usable=gap.usable)
+
+
+@for_client.register(ZPool)
+def _for_client_zpool(zpool, *, min_size=0):
+    return types.ZPool(
+        pool=zpool.pool,
+        mountpoint=zpool.mountpoint,
+        zfses=[for_client(zfs) for zfs in zpool.zfses],
```

Comment:
Er. there's no for_client implementation for Zfs objects yet?

Reply:
Done